### PR TITLE
SOC-1930 | Tracking mobile / desktop contributions on Discussions

### PR DESCRIPTION
**Pull request by bkoval**
## Links

* http://wikia-inc.atlassian.net/browse/SOC-1930

## Description

Tracking of Posts and Replies creation in discussions, divided between mobile and desktop. Tracks to:

1) GA (sampled on 10%)
2) Internal tracker (_beacon)
3) Clickstream (soon to be forwarded to new data warehouse - https://wikia-inc.atlassian.net/browse/MAIN-6592)

## Reviewers

@rafalkalinski @armonr

### CODE REVIEW

**Comment by rafalkalinski** [comment]
LGTM!

**Comment by kvas-damian** [comment]
I don't like the idea of using our custom solution for sending data into new datawarehouse using clck stream. If this the way we want go use for entire platform please create new tracking module in: https://github.com/Wikia/mercury/tree/SOC-1930/front/common/modules/Trackers and make into `mercuryTrack()`.

File: front/main/app/utils/discussion-tracker.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+/**
+ * @param {object} gaContext
+ *
+ * @returns {object}
+ */
+function getClickstreamEvent(gaContext) {
+	return {
+		timestamp: new Date().getTime(),
```
Comment (armonr) [inline front/main/app/utils/discussion-tracker.js]: Does this handle Timezone?

**Comment by armonr** [comment]
Besides comments above and conflict, :+1:

**Comment by rafalkalinski** [comment]
LGTM

**Comment by kvas-damian** [comment]
LGTM

**Pull request merged**